src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,79 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.InvalidDataException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    public User getDefaultValidUser() {
+        User user = new User();
+        user.setLogin("validLogin");
+        user.setId(38628L);
+        user.setPassword("unuwjf8j2");
+        user.setAge(22);
+        return user;
+    }
+
+    @Test
+    void registrationDataIsValid() throws InvalidDataException {
```

Comment:
all tests should follow the same naming convention mentioned in the common mistakes:
`<methodUnderTheTest>_<parametersThatItRecieves>_<Ok/notOk>`
example: `register_nullLogin_notOk`
check out other point from it too